core/src/main/scala/design/hamu/Main.scala

```diff
@@ -1,8 +1,46 @@
 package design.hamu
+import scala.io.StdIn.readLine
 
 object Main {
 
   def main(args: Array[String]): Unit = {
-    println(Helloworld.greeting)
+    println("Welcome to tic tac toe friends")
+    println("Enter Player 1's name (X)")
+    val playerOneName: String = readLine()
+    println("Enter Player 2's name (O)")
+    val playerTwoName: String = readLine()
+    // Create our player pieces
+
+    val game = new Game(Board.initFreshBoard, true, 'X')
+    println("Possible board choices are")
+    println("0|1|2")
+    println("-----")
+    println("3|4|5")
+    println("-----")
+    println("6|7|8")
+
+    while(game.isActive) {
+      if (game.turn == 'X') {
```

Comment:
Not sure if this fork in logic is necessary since everything is the same except playerName and character.